website/src/pages/index.tsx

```diff
@@ -76,7 +82,11 @@ export default function Home(): ReactNode {
   return (
     <Layout
       title={`${siteConfig.title}`}
-      description="Scaffold - The Stellar app lifecycle streamlined">
+      description={translate({
+        id: "homepage.mainMessage",
```

Comment:
I don't see any translations in place for all these ids. Could you add them, even if they're in English for now to make it easier to update them for a 1st-time contributor later?

Reply:
oh these are in the `code.json`. Or I missed some? I will double check 👍